tensorrt_llm/_torch/visual_gen/models/ltx2_retake/ltx2_retake_core/media_io.py

```
@@ -0,0 +1,212 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025–2026 Lightricks Ltd.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)